src/main/java/org/example/utils/Properties.java

```diff
@@ -0,0 +1,32 @@
+package org.example.utils;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+
+@Getter(AccessLevel.PUBLIC)
```

Comment:
By default, when we use the Lombok’s @Getter and @Setter annotations the getters and setters will be created with public access modifier so you don't need to explicitly set AccessLevel.PUBLIC